setup.cfg

```
@@ -1,6 +1,85 @@
-[bdist_rpm]
-packager = Sylvain Thenault  <sylvain.thenault@logilab.fr>
-provides = pylint
+[metadata]
+name = pylint
+license = GPL-2.0-or-later
+license_file = COPYING
+description = python code static checker
+keywords = static code analysis linter python lint
+long_description = file: README.rst
+long_description_content_type = text/x-rst; charset=UTF-8
```

```suggestion
long_description_content_type = text/x-rst
```
`UTF-8` is already the default if omitted. https://packaging.python.org/specifications/core-metadata/#description-content-type

---

👍